testgen/src/main/scala/TriangleTestGenerator.scala

```diff
@@ -0,0 +1,37 @@
+import java.io.File
+
+import testgen.TestSuiteBuilder._
+import testgen._
+
+object TriangleTestGenerator {
+  def main(args: Array[String]): Unit = {
+    val file = new File("src/main/resources/triangle.json")
+
+    def mapArgToString(arg: Any): String = {
+      arg match {
+        case vals: List[_] => vals.mkString(", ")
+        case _ => throw new IllegalStateException
+      }
+    }
+
+    def sutArgs(parseResult: CanonicalDataParser.ParseResult, argNames: String*): String =
+          argNames map (name => mapArgToString(parseResult(name))) mkString
+
+    def fromLabeledTest(argNames: String*): ToTestCaseData =
```

Comment:
I'm fine with using a stateless object here, it's not that it requires a lot of fancy logic for which an object could be useful.